### DISCUSSION TITLE
Problem to upload files larger than 10MB on docker-based config

### DISCUSSION THREAD

**Discussion by EgonHeuson** [Chevereto/Chevereto-Free#179] [Help needed]
Hello all,
I just installed Chevereto using the [nmtam/chevereto](https://hub.docker.com/r/nmtan/chevereto/) docker instance. I've set up a php.ini file to overpass the 2MB file size limitation in upload, but even if I set up 100MB, and if the parameter is well recognized by the server (see capture below), it says that the images that are over 10MB are to big. If I upload an image of 9.8MB no problem, 10.2MB doesn't work. I also tried to set up 20MB as limit in the php.ini file and it's the exact same size limit I get. Any idea? 

<img width="663" alt="2022-08-02_19-47-37" src="https://user-images.githubusercontent.com/32910291/182440513-bcffe3a8-f07c-42f0-ad14-28f5e3ac68ff.png">

<img width="957" alt="2022-08-02_19-40-17" src="https://user-images.githubusercontent.com/32910291/182440364-f720a017-3c41-4473-a877-e10a8e8ca52c.png">

**Comment by rodber** [comment]
https://github.com/rodber/chevereto-free/pkgs/container/chevereto-free

**Reply by EgonHeuson** [reply]
Hello, thanks but I do not understand?

**Reply by TheRealBarenziah** [reply]
I believe what rodber is saying is that you didn't install Docker using the official image (which he linked) but a third party one, for which this is the wrong place to ask support for.

You should try using the [official one](https://github.com/rodber/chevereto-free/blob/1.6/httpd-php.Dockerfile#L57), which happen to have a `CHEVERETO_UPLOAD_MAX_FILESIZE` env variable

Also, tinkering with config files (php.ini) within the container itself shouldn't work in most cases since those are usually read once at application (container) startup. You should have better luck with the dedicated env variable

Good luck :+1:

**Reply by EgonHeuson** [reply]
Ok I see. Thanks! Anyway, as far as I can read, this won't be maintained in few month right? 
About the php.ini, you're right, but I prune and recreate the image each time I perform a modification ;-)

**Reply by rodber** [reply]
That the software won't get future maintenance doesn't means that it will stop working that quickly. If you use Docker, you will be able to run it for years as it provides all the infra.
